about-pses/about-pses.md

```diff
@@ -3,38 +3,35 @@
 ## Introduction
 
 We need to solve problems with other people, whether it's in a whiteboard interview for a new job or talking through a hard problem with a teammate.
+- In both of these scenarios, the most important thing is to communicate and display our thought processes while coding. 
 
-In both of these scenarios, the most important thing is to communicate and display our thought processes while coding. Interviewers and teammates look for the ability to think like a programmer.
-
-But how do we demonstrate that we think like a programmer while solving problems?
-
-We demonstrate how we think like a programmer by:
-
+Interviewers and teammates look for the ability to think like a programmer. But how do we demonstrate that we think like a programmer while solving problems? We demonstrate how we think like a programmer by:
 - Sharing our understanding of the problem
 - Asking the right questions
 - Talking about coding concepts in multiple different ways
 - Solving small coding problems while working towards a bigger solution
 
 ## What Are PSEs?
 
-Problem-Solving Exercise assignments (PSEs) are assignments to practice problem-solving skills.
+Problem-Solving Exercise assignments (PSEs) are assignments to practice problem-solving skills that we will use in interviews and when communicating about code with others.
 
-The exercises will touch on problem-solving skills on coding problems. The format has three parts:
+Interviews are conducted in a setting that enables discussion and thinking out loud. As a result, the practice in PSEs will be an imperfect approximation, but the act of thinking about and rehearsing how we can word these ideas is valuable preparation. In these exercises, we will write down what would mostly be talked through in an actual interview. 
+- During an interview, writing some of the discussion points down can still be useful but our notes there would likely not be to the same level of detail as the responses for PSEs.
```

Comment:
I feel like this might be a good time to mention something along the lines of us asking them to interact with AI tools during the PSE process but that they will likely not have access to those tools in an interview.
```suggestion
- During an interview, writing some of the discussion points down can still be useful but our notes there would likely not be to the same level of detail as the responses for PSEs.
    - For example, our PSEs will ask you to use AI tools to supplement your critical thinking . You will likely not have access to AI tools in an Interview.
```

Reply:
I like that, how do you feel about it being rephrased slightly as a second bullet point without further indentation?

```suggestion
- During an interview, writing some of the discussion points down can still be useful but our notes there would likely not be to the same level of detail as the responses for PSEs.
- Additionally, our PSEs will ask you to use AI tools to further explore and supplement your initial findings. You will likely not have access to AI tools in an Interview.
```